GrowthViz-adults.py

```diff
@@ -299,13 +270,13 @@ def handle_selection_change(_event, _widget):
             wt_df=fixed(wt_percentiles), bmi_df=fixed(bmi_percentiles), ht_df=fixed(ht_percentiles))
 
 
-# In[22]:
+# In[ ]:
 
 
 obs_wbmi[obs_wbmi['subjid'] == 2868]
```

Comment:
It seems like something's missing here. Did we lose something? There should be some explanatory text before `all_charts()`, and this one filter by subjid line isn't explained.

Might need to ticket and resolve this separately.

Reply:
- [ ] @dchud will ticket and resolve using file history